app/src/main/res/drawable/splash_screen.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8"?>
+<layer-list xmlns:android="http://schemas.android.com/apk/res/android">
+    <item
+        android:drawable="@color/white"/>
+
+    <item>
+        <bitmap
+            android:gravity="center"
+                android:src="@drawable/splash_logo"
```

Comment:
nit: Make the spacing between src and gravity consistent. Looks like we just need to indent back a tab?

Reply:
Reminder to follow up on this.